exec/cnex/dictionary.c

```diff
@@ -79,12 +79,49 @@ Dictionary *dictionary_createDictionary(int64_t size)
     d->len = 0;
     d->max = size;
     d->data = malloc(d->max * sizeof(struct tagTDictionaryEntry));
-    if (!d->data) {
+    if (d->data == NULL) {
         fatal_error("Could not allocate space for %d dictionary entries.", d->max);
     }
+    //for (int64_t i = 0; i < d->len; i++) {
```

Comment:
What is the purpose of this commented-out code? `d->len` is 0 from a few lines above, so it doesn't make sense here.

Reply:
Removed.  This was an attempt to allocate all of the dictionary records as they were created, but that created a memory management nightmare.